test/test_output.jl

```diff
@@ -1,8 +1,10 @@
 macro test_save(fmt)
     quote
-        let pl = plot(1:10), fn = tmpname() # fn is an AbstractPath from FilePathsBase.jl
+        let pl = plot(1:10), fn = tempname(), fp = tmpname() # fp is an AbstractPath from FilePathsBase.jl
             getfield(Plots, $fmt)(pl, fn)
             getfield(Plots, $fmt)(fn)
+            getfield(Plots, $fmt)(fp)
+            # ...
```

Comment:
To be finished ...

Reply:
Oh yeah, my bad

Reply:
Maybe I wasn't that clear enough (reviewing on my phone is less easier than on a pc ;))